dashboard/pkg/epinio/models/services.js

```diff
@@ -94,4 +95,11 @@ export default class EpinioServiceModel extends EpinioNamespacedResource {
   bulkRemove(items, opt) {
     return bulkRemove(items, opt);
   }
+
+  async details() {
```

Comment:
this is just fetching the service resource?

if we need an update, it should be via `epinio/find` with an opt include `force: true`, then the latest version will be in the store

Reply:
Replaced with `store` function.